tests/py/test_email.py

```diff
@@ -108,6 +116,17 @@ def test_verify_email(self):
         actual = Participant.from_username('alice').email
         assert expected == actual
 
+    def test_email_verification_is_backwards_compatible(self):
+        """Test email verification still works with unencoded email in verification link.
+        """
+        self.hit_email_spt('add-email', 'alice@example.com')
+        nonce = self.alice.get_email('alice@example.com').nonce
+        url = '/alice/emails/verify.html?email=alice@example.com&nonce='+nonce
+        self.client.GET(url, auth_as='alice')
```

Comment:
Our `auth_as` is different from Gratipay's, it takes a `Participant` object, not a username.